docs/virtual_machines/liveness_and_readiness_probes.md

```diff
@@ -200,13 +223,12 @@ Operating System is focused on. One can configure the `i6300esb` watchdog
 device:
 
 ```yaml
----
 apiVersion: kubevirt.io/v1
 kind: VirtualMachineInstance
 metadata:
   labels:
-    special: vmi-with-watchdog
-  name: vmi-with-watchdog
+    special: vmi-with-watchdog-new
```

Comment:
Do we need the suffix `-new`?

Reply:
No. My bad. I forgot to remove it after testing with the old and the new version of the yaml. Fixed!